icu4c/source/test/testdata/rbbitst.txt

```diff
@@ -1525,6 +1525,31 @@ Bangkok)•</data>
 #
 ####################################################################################
 
+# -u-dx (exclude script)
+#<locale th
+
+
+#<line>
+
+<locale sss@dx=thai>
+<line>
+# Should no longer break at the dictionary points - it's not Thai language
+# Short Test
+<data>•โอํน• อะไป •จู่วาม •โล่น•</data>
```

Comment:
I think this should be
````
<data>•โอํน •อะไป •จู่วาม •โล่น•</data>
```
the line break should happen after the ' ' not before.